packages/mock-block-dock/package.json

```diff
@@ -26,7 +26,7 @@
   "scripts": {
     "build": "yarn clean && yarn build:cjs && yarn build:esm",
     "build:cjs": "tsc --skipLibCheck --project tsconfig.build.cjs.json",
-    "build:dev": "yarn clean && cross-env NODE_ENV=development webpack --mode development",
+    "build:dev": "yarn clean && cross-env NODE_ENV=development webpack --mode development --config=dev/webpack.config.js",
```

Comment:
Fixes this warning:

```
[webpack] Module not found: Error: Can't resolve './src' in '/path/to/repo/packages/mock-block-dock'
[webpack] resolve './src' in '/path/to/repo/packages/mock-block-dock'
[webpack]   using description file: /path/to/repo/packages/mock-block-dock/package.json (relative path: .)
[webpack]     Field 'browser' doesn't contain a valid alias configuration
[webpack]     using description file: /path/to/repo/packages/mock-block-dock/package.json (relative path: ./src)
[webpack]       no extension
[webpack]         Field 'browser' doesn't contain a valid alias configuration
[webpack]         /path/to/repo/packages/mock-block-dock/src is not a file
[webpack]       .js
[webpack]         Field 'browser' doesn't contain a valid alias configuration
[webpack]         /path/to/repo/packages/mock-block-dock/src.js doesn't exist
[webpack]       .json
[webpack]         Field 'browser' doesn't contain a valid alias configuration
[webpack]         /path/to/repo/packages/mock-block-dock/src.json doesn't exist
[webpack]       .wasm
[webpack]         Field 'browser' doesn't contain a valid alias configuration
[webpack]         /path/to/repo/packages/mock-block-dock/src.wasm doesn't exist
[webpack]       as directory
[webpack]         existing directory /path/to/repo/packages/mock-block-dock/src
[webpack]           using description file: /path/to/repo/packages/mock-block-dock/package.json (relative path: ./src)
[webpack]             using path: /path/to/repo/packages/mock-block-dock/src/index
[webpack]               using description file: /path/to/repo/packages/mock-block-dock/package.json (relative path: ./src/index)
[webpack]                 no extension
[webpack]                   Field 'browser' doesn't contain a valid alias configuration
[webpack]                   /path/to/repo/packages/mock-block-dock/src/index doesn't exist
[webpack]                 .js
[webpack]                   Field 'browser' doesn't contain a valid alias configuration
[webpack]                   /path/to/repo/packages/mock-block-dock/src/index.js doesn't exist
[webpack]                 .json
[webpack]                   Field 'browser' doesn't contain a valid alias configuration
[webpack]                   /path/to/repo/packages/mock-block-dock/src/index.json doesn't exist
[webpack]                 .wasm
[webpack]                   Field 'browser' doesn't contain a valid alias configuration
[webpack]                   /path/to/repo/packages/mock-block-dock/src/index.wasm doesn't exist
[webpack] 
[webpack] webpack 5.75.0 compiled with 1 error in 28 ms
```

`webpack-cli` does not look into `dev` folder by default. This applies to older versions too.